python/hsfs/feature_view.py

```diff
@@ -132,7 +132,7 @@ def init_batch_scoring(
         """Initialise and cache parametrized transformation functions.
 
         # Arguments
-            training_dataset_version: int, optional. Default to be 1. Transformation statistics
+            training_dataset_version: int, optional. Default to be None. Transformation statistics
```

Comment:
TD version cannot be default to 1 here because the backend will try to retrieve the td metadata and get the extra filter. If we set the default to 1, it will cause 404 error. @davitbzh any comments?